src/lib/components/data-table/column-helpers.ts

```
@@ -0,0 +1,48 @@
+import type { ColumnDef } from '@tanstack/table-core';
+import { renderComponent } from '$lib/components/ui/data-table/index.js';
+import { DataTableActions, DataTableSortButton, DataTableExpandButton } from './index.js';
```

ESLint

[error] 'DataTableExpandButton' is defined but never used.

(@typescript-eslint/no-unused-vars)